client/src/App.js

```diff
@@ -1,26 +1,26 @@
-import React from 'react';
-import logo from './logo.svg';
-import './App.css';
+import React from "react";
+import "./App.css";
+import { BrowserRouter, Switch, Route } from "react-router-dom";
+import GameThread from "./pages/GameThread";
```

Comment:
You might want to put all of these page imports into a single file and import that as an object so you don't have a ton of "import page 1, import page 2, ..."

Reply:
agreed!